nixos/modules/services/hardware/pcscd.nix

```diff
@@ -5,6 +5,10 @@ with lib;
 let
   cfgFile = pkgs.writeText "reader.conf" config.services.pcscd.readerConfig;
 
+  package = if config.security.polkit.enable
+              then pkgs.pcscliteWithPolkit
```

Comment:
```suggestion
              then (pkgs.pcsclite.override { polkitSupport = true; })
```

Reply:
Will this be built and distributed in binary cache?

Reply:
The [answer](https://matrix.to/#/%23dev%3Anixos.org/%24FZSSO-Q7LyfhpxF1kJR6eQBm5ZOog9_I2cmZRYM8PI4) is no.

Reply:
Can't we set the polkit support by default to on and disable it when polkit is disabled? Then most people get a cache hit and we don't need to add another top-level entry just for caching.